src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -735,7 +735,7 @@ public void execute() throws GitException, InterruptedException {
 
                 args.add(fastForwardMode);
                 args.add(rev.name());
-                
+
```

Comment:
Please don't make whitespace changes.  There are many pull requests which have been intentionally closed in hopes of reopening them later as capacity and priority allows.  When white space corrections are made, those pending changes become more complicated to evaluate due to conflicts in white space changes.